tests/suites/test_suite_pk.function

```diff
@@ -424,14 +434,6 @@ void valid_parameters()
     TEST_ASSERT(mbedtls_pk_get_len(NULL) == 0);
     TEST_ASSERT(mbedtls_pk_can_do(NULL, MBEDTLS_PK_NONE) == 0);
 
-    TEST_ASSERT(mbedtls_pk_sign_restartable(&pk,
```

Comment:
Why are you removing this? (Not saying it's wrong, but if it's correct, the reason's not obvious to me.)

Reply:
Ah, indeed, thanks!